docs/topics/modules/creating_new_space.adoc

```diff
@@ -1,14 +1,15 @@
 [id="creating_new_space-{context}"]
 = Creating a new space
 
+In {osio}, the first step for any new project is to create a new space.
+
 // for hello-world
 ifeval::["{context}" == "hello-world"]
-In {osio}, the first step for any new project is to create a new space. You can then add users as collaborators for the space and all the work is tracked within the space using work items. Each space must have a unique name.
+ You can then add users as collaborators for the space and all the work is tracked within the space using work items. Each space must have a unique name.
```

Comment:
@Preeticp sorry, I just noticed this today when building the doc. It looks like this is rendering as some weird block quote. Please have a look since you know the changes best.

Reply:
Specifically, this is what I'm seeing in the hello world instance of this module in the GS doc:

![gsg_quote](https://user-images.githubusercontent.com/926572/39557069-52803a10-4ec8-11e8-82ab-6c3f5593db1f.png)

Reply:
Strange, was rendering fine when I built it, before I pushed. I will check and resolve this.